contrib/gumbo/template.py

```diff
@@ -0,0 +1,37 @@
+pkgname = "gumbo"
+pkgver = "0.12.1"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_gen = ["./autogen.sh"]
+make_cmd = "gmake"
+hostmakedepends = [
+    "bash",
+    "gmake",
+    "gm4",
+    "pkgconf",
+    "automake",
+    "autoconf",
+    "gtest-devel",
+    "libtool",
```

Comment:
remove autoconf, put gtest-devel in makedepends  

it also seems bash is not needed

Reply:
also sort the list, same below